test/fixtures/rules/std_rule_cases/L025.yml

```diff
@@ -284,16 +312,3 @@ test_fail_snowflake_flatten_function:
   configs:
     core:
       dialect: snowflake
-
-
-test_pass_snowflake_values:
-  # Tests a fix for issue 3301.
-  pass_str: |
-    SELECT
-      thing_1
-      , thing_2
-    FROM VALUES
-      ( 'foo', 'bar')
-      , ( 'foo', 'bar')
-      my_table_alias(thing_1, thing_2)
-    ;
```

Comment:
I moved this existing test case so it'd be nearer the various test cases involving `VALUES` clauses in different dialects -- seems better organized to do this, since this PR adds dialect-specific logic for how `VALUES` clauses are handled.